### PR TITLE
Fix the issue of blocking merge when making it required

### PR DESCRIPTION
**Pull request by ydah**
Fix: https://github.com/rubocop/rubocop-capybara/pull/39#issuecomment-1509796770
This PR change the control from `paths` to `if`.

______________________________________________________________________

Before submitting the PR make sure the following are checked:

- [x] Feature branch is up-to-date with `main` (if not - rebase it).
- [x] Squashed related commits together.
- [-] Added tests.
- [-] Updated documentation.
- [-] Added an entry to the `CHANGELOG.md` if the new code introduces user-observable changes.
- [x] The build (`bundle exec rake`) passes (be sure to run this locally, since it may produce updated documentation that you will need to commit).

### CODE REVIEW

File: .github/workflows/mdformat.yml [hunk 1]
```diff
@@ -12,8 +10,14 @@ jobs:
...
     steps:
       - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v35
       - name: Mdformat
         uses: ydah/mdformat-action@main
+        if: contains(steps.changed-files.outputs.all_changed_files, '.md')
```
Comment (ydah) [inline .github/workflows/mdformat.yml:20]: memo) If we are going to go this far, perhaps we should simply remove filtering by `paths` or make it not required. WDYT?

Comment (pirj) [reply .github/workflows/mdformat.yml:20]: I suggest running this all the time with no path filtering, no checks if any files were changed etc, and keep it required.
Are there any downsides to this approach?

Comment (bquorning) [reply .github/workflows/mdformat.yml:20]: That’s what I would suggest as well.

Comment (ydah) [reply .github/workflows/mdformat.yml:20]: It certainly sounds like it should always be done.
I will open the next PR and try to delete this one.
- https://github.com/rubocop/rubocop-capybara/pull/41

**Pull request closed without merging**